src/containers/StaffManagement/StaffManagment.test.tsx

```diff
@@ -30,7 +30,7 @@ test('it should have a help link', async () => {
   // loading is show initially
   expect(screen.getByText('Loading...')).toBeInTheDocument();
   await waitFor(async () => await new Promise((resolve) => setTimeout(resolve, 0)));
-  const helpButton = screen.getByTestId('helpButton');
+  const helpButton = await screen.getByTestId('helpButton');
```

Comment:
I've updated this test case, wrapped code under `await waitFor` in the Shamoon branch.
note: **await** is used with **findBy**

Reply:
I don't see any open PR's., what branch?

> note: await is used with findBy

Not sure what you mean here. Let's discuss tomorrow.

Reply:
Fixed.